data/human/hails.txt

```diff
@@ -2082,6 +2082,88 @@ phrase "friendly civilian"
 		"Zug. I used to watch the militia ships fight against the pirate ships in the sky from my porch."
 		"New Holland. It gets too boring there, though, so I fly the stars instead."
 		"Bounty. I bought my first ship when the manufacturing industry started to develop there."
+		
+phrase "pyramid scheme"
+	word
+		"Magellan "
+		"Dehydrated "
+		"Pontificated "
+		"General "
+		"Sex Sex Sex "
+		"Sacrosanct "
+		"Liberated "
+		"Literal "
+		"Family "
+	word
+		"Dynamix"
+		"Unlimited"
+		"Orbital"
+		"Hyper-real"
+		"Gigalight"
+		"Traders"
+		"Voidcraft"
+		"Whispers"
+
+phrase "friendly civilian"
+	word
+		"Hi! Good to see you, uh, "
+		"Well, if it isn't "
+		"Hi there, uhm, "
+		"I've been trying to get a hold of you, "
+	word
+		"Captain <Last>. "
+	word
+		"I'm living the dream as a CEO of my own company. "
+		"I used to be just like you, but now I'm really somebody! "
+		"Don't you want to be your own boss? "
+		"Everybody is talking about this slick new fad! "
+	word
+		"You could get rich selling for me!"
+		"Get in on the ground floor!"
+		"We're about to go orbital, you should sign up while you still can!"
+	word
+		" "
+	word
+		"Find me "
+		"You can get me "
+		"Call me "
+		"Drop me a line "
+		"Buzz me "
+	word
+		"at ${name}@${pyramid scheme}."
+
+phrase "friendly civilian"
+	word
+		"All I want is one day without "
+		"I just bought this ship and it already has "
+		"The warranty for my ship expired a week ago, and wouldn't you know it, I just found "
+	word
+		"a fuel leak! "
+		"a loose coupling! "
+		"a fault in my nav-array! "
+		"a power cascade! "
+		"an anti-matter containment failure! "
```

Comment:
```suggestion
		"a deuterium containment failure! "
```

Reply:
Deuterium isn't really dangerous unlike antimatter.

Reply:
Could change it to oxygen. You got a hull breach and air leaking out of your ship, that's a bad sign.